content/blog/2020/12/Camel37-Whatsnew/index.md

```diff
@@ -0,0 +1,196 @@
+---
+title: "Apache Camel 3.7 What's New"
+date: 2020-12-17
+authors: [davsclaus]
+categories: ["Releases"]
+preview: Details of what we have done in the Camel 3.7 release.
+---
+
+Apache Camel 3.7 LTS has just been released.
+
+This is a LTS release which means we will provide patch releases for one year.
+The next planned LTS release is 3.10 scheduled towards summer 2021.
+
+
+## So what's in this release?
+
+This release introduces a set of new features and noticeable improvements that we will cover in this blog post.
+
+### Pre compiled languages
+
+We continued our avenue of making Camel faster and smaller.
+This time we focused on the built-in Simple scripting language.
+
+First we added the [jOOR language](/components/latest/languages/joor-language.html). jOOR is a small Java tool
+for performing runtime compilation of Java source code in-memory. It has some limitations but generally works well for small
+scripting code (requires Java 11 onwards).
+
+Then we worked on compiled simple.
+
+### Compiled Simple
+
+The [csimple language](https://camel.apache.org/components/latest/languages/csimple-language.html)
+is parsed into regular Java source code and compiled together with all the other source code,
+or compiled once during bootstrap via jOOR.
+
+{{< image "csimple-compiled.png" "Compiled Simple Language" >}}
+
+In a nutshell where compiled simple excels over simple language is primary around dynamic OGNL method calls.
```

Comment:
I had to read several times to understand this sentence.

What about something like:
In a nutshell, `Compiled Simple` excels primarily over simple language when using dynamic Object-Graph Navigation Language ([OGNL](https://en.wikipedia.org/wiki/OGNL) ) method calls..

and wtih a link to